src/integration/pushResult.js

```diff
@@ -45,17 +45,21 @@ function addCommit(src, msg){
 //pulling from remote repo
```

Comment:
Also, add some message after pulling/pushing is done because this generates a lot of warnings and it is hard to read from logs